contracts/src/modules/HarvesterPlugin.sol

```diff
@@ -27,9 +34,12 @@ contract DummyModule is BaseModule {
   // address (SafeProtocolManager)
   address manager;
 
-  // address (relayer: keeper, gelato, AA)
+  // address (relayer: cl keeper, gelato, AA)
   address public relayer;
 
+  /// @notice Address of the safes with plugin enabled
+  EnumerableSet.AddressSet internal safes;
```

Comment:
maybe also something that would fit better in the registry?

Reply:
although it is probably inefficient from a gas perspective to constantly have to call the registry for this.

Reply:
yes, it will be suboptimal from the gas pov and also i think is better to isolate plugis<>safes hook into it, rather than involving registry for this action also